app/repository/sql/pipeline.sql

```diff
@@ -0,0 +1,244 @@
+                slug.created DESC,
+                slug.ctid DESC ),
+                unique_doc_slugs AS (   SELECT
+                    DISTINCT
+                        ON (slug.family_document_import_id)   slug.family_document_import_id,
+                        slug.created,
+                        slug.name
+                FROM
+                    (     SELECT
+                        slug.family_document_import_id AS "family_document_import_id",
+                        Count(*) AS count
+                    FROM
+                        slug
+                    WHERE
+                        slug.family_document_import_id IS NOT NULL
+                    GROUP BY
+                        slug.family_document_import_id
+                    HAVING
+                        Count(*) = 1   ) non_duplicates
+                left join
+                    slug
+                        ON non_duplicates.family_document_import_id = slug.family_document_import_id
+                ORDER BY
+                    slug.family_document_import_id DESC,
+                    slug.created DESC,
+                    slug.ctid DESC   ),
+                    most_recent_doc_slugs AS (   SELECT
+                        deduplicated_doc_slugs.family_document_import_id   AS "family_document_import_id",
+                        deduplicated_doc_slugs.created,
+                        deduplicated_doc_slugs.name
+                    FROM
+                        deduplicated_doc_slugs
+                    UNION
+                    ALL   SELECT
+                        unique_doc_slugs.family_document_import_id AS "family_document_import_id",
+                        unique_doc_slugs.created,
+                        unique_doc_slugs.name
+                    FROM
+                        unique_doc_slugs
+                    ORDER BY
+                        family_document_import_id DESC,
+                        created DESC   ), event_dates AS (   SELECT
+                        family_event.family_import_id AS family_import_id,
+                        Min(CASE
+                            WHEN family_event.event_type_name='Passed/Approved' THEN     family_event.DATE::DATE
+                            ELSE family_event.DATE::DATE
+                        END) published_date,
+                        Max(family_event.DATE::DATE) last_changed
+                    FROM
+                        family_event
+                    GROUP BY
+                        family_import_id )  SELECT
+                        f.title AS "family_title",
+                        p.title AS "physical_document_title",
+                        f.description AS "family_description",
+                        CASE
+                            WHEN f.family_category IN ('UNFCCC',
+                            'MCF') THEN Upper(f.family_category::text)
+                            ELSE Initcap(f.family_category::text)
+                        END "family_category",
```

Comment:
Appreciate this isnt going tio be dealt with here, but might be worth a ticket for revisiting. Although maybe its fine because we arent keeping family_category long term (I think?)

Reply:
Yeah this is being removed in the near-ish future. I'll raise a ticket though :)

https://linear.app/climate-policy-radar/issue/PDCT-1609/remove-family-category-from-the-backend-in-favour-of-genre